srcpkgs/itch/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.18.0
+revision=1
+archs="i686 x86_64"
+wrksrc="${pkgname}-setup-${version}"
+build_style=go
+go_import_path="github.com/itchio/itch-setup"
+hostmakedepends="git pkg-config"
+makedepends="gtk+3-devel"
+short_desc="Itch.io indie games client for Linux"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="MIT"
+homepage="https://itch.io/"
+distfiles="
+ https://github.com/itchio/itch-setup/archive/v${version}.tar.gz
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon16.png
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon32.png
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon48.png"
+checksum="
+ 96d3bff9b200dc5c03af8e46d0805a6f4dc504d6281b2956dcedda931849c0dc
+ c1e7154020c847cd5cd75d544a7e5dbe8bd495fe87a664afd24c4a952a7e8a78
+ e38c4a04b2ced31d2a13b72f310e069beff73a7cf887a4cbde57fa7b19c67fab
+ 5f38d0b884053b10d856151931b7699723c4a28cf14622c95bc6022bec391c3f"
+skip_extraction="icon16.png icon32.png icon48.png"
+
+do_build() {
+	cd "$GOSRCPATH"
+	go get -p "$XBPS_MAKEJOBS" -x -tags "${go_build_tags}" -ldflags "${go_ldflags}" ${go_package}
+}
+
+do_install() {
+	vbin ${GOPATH}/bin/itch-setup
+	vbin ${FILESDIR}/itch
+	vmkdir usr/share/applications
+	vmkdir usr/share/icons/hicolor/16x16/apps
+	vmkdir usr/share/icons/hicolor/32x32/apps
+	vmkdir usr/share/icons/hicolor/48x48/apps
+	vinstall ${FILESDIR}/io.itch.itch.desktop 0644 usr/share/applications
+	cp ${XBPS_SRCDISTDIR}/itch-${version}/icon16.png ${DESTDIR}/usr/share/icons/hicolor/16x16/apps/itch.png
+	cp ${XBPS_SRCDISTDIR}/itch-${version}/icon32.png ${DESTDIR}/usr/share/icons/hicolor/32x32/apps/itch.png
+	cp ${XBPS_SRCDISTDIR}/itch-${version}/icon48.png ${DESTDIR}/usr/share/icons/hicolor/48x48/apps/itch.png
```

Comment:
Please use vinstall here as well.

Reply:
The vinstall command does not work for this, as the files are downloaded separately and then moved to a folder.